Userland/Applications/Browser/CookieJar.cpp

```diff
@@ -75,6 +75,17 @@ void CookieJar::dump_cookies() const
     dbgln("{}", builder.build());
 }
 
+Vector<Web::Cookie::Cookie> CookieJar::want_cookies() const
```

Comment:
`want_cookies` is a bit of a weird function name. `get_all_cookies` or something like that would be better I think.

We should also reserve capacity in the `cookies` vector up front so we don't keep reallocating in the `for` loop. That `for` loop also doesn't need braces since it's a single line, and should use east-const instead of west-const:

```c++
Vector<Web::Cookie::Cookie> CookieJar::get_all_cookies() const
{
    Vector<Web::Cookie::Cookie> cookies;
    cookies.ensure_capacity(m_cookies.size());

    for (auto const& cookie : m_cookies)
        cookies.unchecked_append(cookie.value);

    return cookies;
}
```

Reply:
Done :)